app/ante/ante.go

```diff
@@ -75,3 +77,10 @@ func NewAnteHandler(
 }
 
 var DefaultSigVerificationGasConsumer = ante.DefaultSigVerificationGasConsumer
+
+// The purpose of this wrapper is to ensure that CheckTxFeeWithGlobalMinGasPrices complies with the ante.TxFeeChecker type.
+func CheckTxFeeWithGlobalMinGasPricesWrapper(paramKeeper paramkeeper.Keeper) ante.TxFeeChecker {
```

Comment:
We actually use this wrapper as a means of passing the paramKeeper into the ante.TxFeeChecker function because it's needed to get the global min gas price

Reply:
yeah I wanted to express that function signatures were otherwise incompatible because we added extra parameter - paramKeeper